textproc/lout/Portfile

```diff
@@ -23,6 +24,13 @@ depends_build   port:zlib
 
 use_configure   no
 
+platform darwin 16 {
+     # On Sierra, system clang and clangs 3.8 and 3.9 fail to make a working binary
+     # see <https://github.com/macports/macports-ports/pull/662>
+     compiler.blacklist-append   {macports-clang-3.[0-9]} clang
+     compiler.fallback-append    {macports-clang-4.0} {macports-clang-5.0}
+}
```

Comment:
There is no reason to enclose plain compiler names inside curly brackets, such as on the fallback line. (I see why you're doing so on the blacklist line, though, to prevent Tcl from interpreting the square brackets.)